### DISCUSSION TITLE
几天了我蓝灯还是无法登录

### DISCUSSION THREAD

**Discussion by pppppppppppppjsjns** [getlantern/lantern-forum-cn#774] [Q&A]
重新登陆不发邮箱验证码
帐号574471458@qq.com
您尾号6381交行信用卡05日00时06分成功消费人民币344.00

**Comment by MaxMKMax** [comment]
@pppppppppppppjsjns 您好. 请您在此提供我支付单据. 我再次帮您提交问题. 不好意思让您久等了. 谢谢您的耐心等待.

您是看能把支付单据发到这里吗? 如果不能就发邮箱给我. 谢谢!

您之前的图被GitHub屏蔽了. 我这里看不到. 还是发邮箱给我吧!

**Reply by zhudamao123** [reply]
就算发了又如何呢，你能解决问题吗？当时说发激活码，我的激活码等了很久了，现在也没等到

**Reply by whpublic2024** [reply]
一样的，我邮件发了一个多星期了，还是不给发。

**Comment by MaxMKMax** [comment]
@pppppppppppppjsjns 
您好. 请您试试看这个验证码:
307074
